doc/Language/subscripts.pod6

```diff
@@ -253,68 +251,74 @@ dimensions>) the subscript is preserved across the slice operation
 
 =head2 Truncating slices
 
-Normally, referring to nonexistent elements in a slice subscript causes the
-output list to contain undefined values (or L<whatever else|
+Referring to nonexistent elements in a slice subscript causes the
+output C<List> to contain undefined values (or L<whatever else|
 #Nonexistent elements> the collection in question chooses to return for
-nonexistent elements). However if the outer object passed to
-(L<one dimension of|#Multiple dimensions>) of a positional subscript is a
-L<Range>, it will be automatically truncated to the actual size of the
-collection:
-
-    my @letters = <a b c d e f>;
-    say @letters[3, 4, 5, 6, 7];  # OUTPUT: «(d e f (Any) (Any))␤»
-    say @letters[3 .. 7];         # OUTPUT: «(d e f)␤»
-
-L<From-the-end|#From the end> indices are allowed as range end-points.
+nonexistent elements):
 
-    =begin code :preamble<my @array;>
-    say @array[*-3 .. *];       # select the last three elements
+    =begin code
+    my  @letters = <a b c d e f>;
+    say @letters[3..7];  # OUTPUT: «(d e f (Any) (Any))␤»
     =end code
 
-A similar thing is done for lazy sequences, but it is often impossible to
-determine whether the sequence is infinite.  Just as often, the first part
-of the sequence is already known, and it would be silly to pretend we
-did not know it.  As a stopgap measure to prevent runaway generation of huge
-lists, a lazy subscript will not truncate as long as it does not have to
-lazily generate values, but once it starts generating values lazily, it
-will stop if it generates a value that points to a nonexistent index.
+This behavior, while at first glance may seem unintuitive, is desirable in
+instances when you want to assign a value at an index in which a defined
+value does not currently exist.
```

Comment:
*[request for change]*

> in which a defined value

"defined" should be dropped here, since [undefined](https://docs.perl6.org/routine/defined) are perfectly acceptable elements as far as existence goes.